.gitmodules

```diff
@@ -22,3 +22,9 @@
 [submodule "ext/catch2"]
 	path = ext/catch2
 	url = https://github.com/catchorg/Catch2.git
+[submodule "ext/backward-cpp"]
+	path = ext/backward-cpp
+	url = https://github.com/bombela/backward-cpp.git
+[submodule "ext/backward"]
+	path = ext/backward
+	url = https://github.com/bombela/backward-cpp.git
```

Comment:
twice ?

Reply:
Yes, it took two attempts =) I've removed one.